src/public/parts/shells/themeManage/public/templates/editor.html

```
@@ -20,7 +20,7 @@ <h3 class="font-bold mb-2 flex items-center gap-2">
 				<img src="https://api.iconify.design/line-md/image.svg" class="w-5 h-5" /> <span data-i18n="themeManage.editor.autoPaletteTitle"></span>
 			</h3>
 			<input type="file" id="img-upload" accept="image/*" class="file-input file-input-bordered file-input-sm w-full mb-2" />
-			<img id="img-preview" class="w-full max-h-48 object-contain rounded-md mb-2 hidden" />
+			<img id="img-preview" class="w-full max-h-48 object-contain rounded-field mb-2 hidden" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)